force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -289,7 +394,19 @@ public with sharing class FlowControlLoopsCollections {
         // The list of money in the wallet, represented in dollars and cents
         List<Decimal> moneyInWallet = new List<Decimal>{0.50, 10, 3.84, 24.60, 9.08, 50, 4.90}; //DO NOT CHANGE
 
-        return null; // Replace null with the variable you used to store the result
+        // Total amount of money
+        Decimal totalMoney = 0;
+        // loop 
+        for (Decimal money : moneyInWallet) {
+            // add the money to the total until it exceeds 40 dollars
+            if(totalMoney < 40){
```

Comment:
This is fine, but personally I would probably do it the other way. Check the if conditional if you need to break